src/PrestaShopBundle/Bridge/AdminController/LegacyControllerBridge.php

```diff
@@ -212,6 +303,7 @@ public function addJS($jsUri, $checkPath = true): void
 
             if ($jsPath && !in_array($jsPath, $this->controllerConfiguration->jsFiles)) {
                 $this->controllerConfiguration->jsFiles[] = $jsPath . ($version ? '?' . $version : '');
+                $this->controllerConfiguration->jsFiles = [$jsPath . ($version ? '?' . $version : '')];
```

Comment:
```suggestion
```
I think this should be removed, because it doesn't stack the js files anymore it will only keep the last one this way

Reply:
Which makes me think we should probably add some unit tests for those two methods addJs and addCss